swagger.yaml

```diff
@@ -200,11 +200,18 @@ definitions:
         description: "List of suggested alternative similar search queries"
         items:
           type: string
+      alternative_suggestions:
+        type: array
+        description: "List of suggested alternative search queries from current search query e.g. return 'uk' and 'economy' and 'uk economy'"
```

Comment:
Might be splitting hairs but is it worth putting the `e.g.` as an `example`?

Reply:
👍